src/dom/DomUtil.js

```diff
@@ -131,7 +131,7 @@ export function removeClass(el, name) {
 	if (el.classList !== undefined) {
 		el.classList.remove(name);
 	} else {
-		setClass(el, Util.trim((` ${getClass(el)} `).replace(` ${name} `, ' ')));
+		setClass(el, (` ${getClass(el)} `).replace(` ${name} `, ' ').trim());

```

Comment:
I think the parentheses around the backticks can be removed. Does the linter not complain about this?

Reply:
You are right, I've removed them. And no, it doesn't look like we have a linting rule configured for this (perhaps we should).